### PR TITLE
Feature/255

### PR DESCRIPTION
**Pull request by jmzwar**
Screen Shot 2023-02-14 at 11 17 42 pm

Screen Shot 2023-02-14 at 11 17 47 pm

### CODE REVIEW

File: v2/lib/useGlobalStakingApr/useGlobalStakingApr.ts [hunk 1]
```diff
@@ -48,7 +48,25 @@ export const useGlobalStakingApr = (enabled: boolean) => {
...
+        isL2,
+        SNXRate,
+        previousWeekRewardsUsd,
+        totalStakedData
+      );
+      const feesApr = calculateGlobalStakingApr(
+        isL2,
+        SNXRate,
+        previousFeePeriodData?.feesToDistribute,
+        totalStakedData
```
Comment (jmzwar) [inline v2/lib/useGlobalStakingApr/useGlobalStakingApr.ts:61]: Nice - super simple.

File: v2/components/RewardsItem/TradingFeesModal.tsx [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+  onClose,
+  feesBurned,
+  sUSDBalance,
+  currentCRatioPercentage,
+  activeDebt,
+  collateralValue,
+}: TradingFeesModalProps) => {
+  const navigate = useNavigate();
+
+  const previousCratio = (100 * collateralValue) / (activeDebt + feesBurned);
```
Comment (bachstatter) [inline v2/components/RewardsItem/TradingFeesModal.tsx:37]: I think this should be `activeDebt - feeBurned`?

Comment (bachstatter) [reply v2/components/RewardsItem/TradingFeesModal.tsx:37]: nope we should add it to debt..

File: v2/components/RewardsItem/TradingFeesModal.tsx [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+              </Text>
+              <Text
+                fontWeight="700"
+                fontSize="12px"
+                lineHeight="16px"
+                as="span"
+                width="33%"
+                textAlign="center"
+              >
+                {`$${formatNumber(activeDebt - feesBurned)}`}
```
Comment (jmzwar) [inline v2/components/RewardsItem/TradingFeesModal.tsx:150]: activeDebt + feesBurned

Comment (bachstatter) [reply v2/components/RewardsItem/TradingFeesModal.tsx]: Yep created a `previousDebt` const

File: v2/components/RewardsItem/TradingFeesModal.tsx [hunk 4]
```diff
@@ -30,11 +32,12 @@ export const TradingFeesModal = ({
...
   activeDebt,
-  collateralValue,
+  collateral,
+  SNXRate,
 }: TradingFeesModalProps) => {
   const navigate = useNavigate();
-
-  const previousCratio = (100 * collateralValue) / (activeDebt + feesBurned);
+  const previousDebtBalance = activeDebt + feesBurned;
+  const previousCratio = calcNewCratioPercentage(collateral, SNXRate, previousDebtBalance);
```
Comment (jmzwar) [inline v2/components/RewardsItem/TradingFeesModal.tsx:40]: nice

**Pull request merged**